src/main/java/core/basesyntax/shop/service/impl/ValidatorImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.shop.service.impl;
+
+import core.basesyntax.shop.service.Validator;
+
+public class ValidatorImpl implements Validator {
+    @Override
+    public boolean test(String s) {
+        return s.matches("(?si)^type,fruit,quantity(?m)"
+                + "(\\n[bspr],(banana|apple|pear),\\d{1,4})+$");
+    }
```

Comment:
What will happen if we will add new type of operation in future? Your code must be universal.
Think about how to use Enum here.